server/routes/users.js

```diff
@@ -1,15 +1,58 @@
 var express = require('express');
 var router = express.Router();
+const jwt = require('jsonwebtoken');
+const dotenv = require('dotenv');
 
 const { PrismaClient } = require('@prisma/client')
 
+dotenv.config();
+
 const prisma = new PrismaClient()
 
 /* GET users listing. */
 router.get('/', async function(req, res, next) {
   const allMyUsers = await prisma.user.findMany()
-  console.log(allMyUsers);
   res.send(allMyUsers)
 });
 
+async function tokenify (user) {
```

Comment:
Not sure if this is testable right now, but we should write a test that verifies it can be decoded in a future story